src/main/charts/confluence/templates/_helpers.tpl

```diff
@@ -332,6 +340,16 @@ For each additional plugin declared, generate a volume mount that injects that l
 {{- end }}
 {{- end }}
 
+{{- define "synchrony.volumes" -}}
+{{ if not .Values.volumes.synchronyHome.persistentVolumeClaim.create }}
+{{ include "synchrony.volumes.synchronyHome" . }}
+{{- end }}
+{{ include "confluence.volumes.sharedHome" . }}
```

Comment:
This is the existing behaviour. We mount shared home to make it easy to pass JDBC driver using `additionalLibraries` value.